plugins/topology/src/utils/resource-utils.ts

```diff
@@ -0,0 +1,40 @@
+import * as _ from 'lodash';
```

Comment:
I suggest importing the specific resources which is utilized in the source code.

```suggestion
import { isEmpty} from 'lodash';
```